### PR TITLE
Add deep investigation docs - Cloud Solutions

### LINKED ISSUE #12: [Deep Investigation] Cloud Solutions

### PR DESCRIPTION
**Pull request by NimperX**

### CODE REVIEW

**Comment by NimperX** [comment]
#12

File: Deep Investigations/Cloud solutions/Details.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+- [References](#references)
+
+##  Introduction
+---
+
+Nextcloud is a fully featured on-premises solution to share and collaborate documents, send and receive emails, manage all audio/video conferencing, manage contacts, manage schedules, etc. Nextcloud is scalable from home office solutions to full sized data center solutions that support millions of users.
+
+Nextcloud permits user and group administration via [LDAP](https://en.wikipedia.org/wiki/Lightweight_Directory_Access_Protocol) or [OpenID](https://en.wikipedia.org/wiki/Lightweight_Directory_Access_Protocol). Content can be shared by defining granular read/write permissions between users and groups. Alternatively, Nextcloud users can create public URLs when sharing files. Logging of file-related actions and disallowing access based on file access rules are also available.
+
+Nextcloud can be extended with plugins to add extra functionalities to the system. The **Nextcloud App store** can be found [here](https://apps.nextcloud.com/), and it contains **more than 200 plugins**. Some of the functionalities that you can add to Nextcloud are as follows.
```
Comment (dhanushkac) [inline Deep Investigations/Cloud solutions/Details.md:28]: Add plugin installation to demo

**Review by dhanushkac — COMMENTED** [review]
Add what will be presented in demo as a list

File: Deep Investigations/Cloud solutions/Details.md (hunk 1 above)
Comment (NimperX) [reply Deep Investigations/Cloud solutions/Details.md:28]: Sure ayye, I'll add this to the Course content demo videos list.

**Review by dhanushkac — APPROVED** [review]

**Pull request merged**